home_work_4/task_2.py

```diff
@@ -0,0 +1,22 @@
+# Задача №2 семинара 4. Hash.
+
+import typing
+
+
+def converter_to_dict(*, a, b):
```

Comment:
Вы немного не так поняли задание. В качестве значения ключа нужно было использовать имя аргумента, но при таком определении функции, вы имя аргумента не получите. Нужно было сделать так.
```suggestion
def converter_to_dict(**kwargs):
```
Либо использовать функцию `locals`